requirements.txt

```
@@ -0,0 +1,4 @@
+dash==2.9
+dash-mantine-components==0.12
+pymongo==4.3
+python-dotenv==1.0
```

The requirements are too restrictive and the installation fails for `pymongo` with "Could not find a version that satisfies the requirement pymongo==4.3".

```suggestion
dash~=2.9
dash-mantine-components==0.12.*
pymongo~=4.3
python-dotenv>=1.0
```